### ISSUE TITLE
Custom hyperparameters using EnvironmentParameters

### ISSUE THREAD

**Issue by icecoolinux** [icecoolinux/arenalab#18]
The user could set custom hyper-parameters which change the enviornment and the agent behavior (reward, enviornment, actions, etc).
Plugins could use these hyper-parameters to train and fit them.
The hyper-params need to have a name and description because an AI algorithm using generative AI can understand what is the goal.

We could use Curriculum Learning using this feature.
